Tasks/GooglePlayReleaseBundle/googleutil.ts

```diff
@@ -66,7 +65,7 @@ export interface GlobalParams {
     params?: PackageParams;
 }
 
-export function getJWT(key: ClientKey): JWT {
+export function getJWT(key: ClientKey): any {
```

Comment:
Same as https://github.com/microsoft/google-play-vsts-extension/pull/246#pullrequestreview-600130735, changed return type from JWT to any. Due to package upgrade, some things must be updated but sorry, I have no idea what to do on this :/